src/Drivers/VarDriver.php

```diff
@@ -17,7 +17,7 @@ public function extension(): string
         return '.php';
     }
 
-    public function load(string $path)
+    public function load(string $path): string
```

Comment:
`load` doesn't necessarily return a string, in the case of the VarDriver, it could be anything since it does a php `include`.